src/types/inventory.ts

```diff
@@ -0,0 +1,39 @@
+export interface ProductInterface  {
+    id: number;
+    uuid: string;
+    name: string;
+    description: string;
+    short_description: string;
+    slug: string;
+    variants: {
+        name: string;
+        description: string;
+        slug: string;
+        attributes: {
+            name: string;
+            value: string;
+        }[];
+    }[];
```

Comment:
**Change**: i really prefer to separate this entity into two more entities can be ProductVariante and ProductAttributes because is the same here and on CreateProductParams just that has an extra property that is warehouse_id that you can extend it and add that extra variable

Reply:
check